working/augmentation-libraries/feature-specification.md

```diff
@@ -210,7 +210,7 @@ import augment 'b.dart';
 augment class C {}
 
 augment void trace() {
-  augment super.trace();
+  augmented.trace();
```

Comment:
This should just be `augmented()`, here and below.

Unlike `super` calls, you don't mention the name of the augmented you're calling, since it's always your own method. (The only exception currently is operators. I'll comment on those below.)

Reply:
yep good catch